Riot/Categories/MXBugReportRestClient+Riot.swift

```diff
@@ -0,0 +1,117 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+import MatrixSDK
+import GBDeviceInfo
+
+extension MXBugReportRestClient {
+    
+    @objc static func vc_bugReportRestClient(appName: String) -> MXBugReportRestClient {
+        guard let client = MXBugReportRestClient(bugReportEndpoint: BuildSettings.bugReportEndpointUrlString) else {
+            fatalError("Could not create MXBugReportRestClient")
```

Comment:
We shouldn't be propagating an unnecessary optional initialiser through the app code. I can make the init `nonnull` in the sdk, probably makes more sense. Work for you?

Reply:
This will work. Or the other way around, we can declare the return value as optional. This way, the caller will know if something wrong happened and have a specific code for this case. I understand you don't want to propagate unnecessary optional initialiser but I'm not sure this will be called many times.